src/server/save_and_load.R

```diff
@@ -32,6 +32,17 @@ writeFilesForDigest <- function(workingSet, spectrumFilesState, surveyAndProgram
             write.csv(surveyAndProgramData$program_data, file = path, row.names = FALSE)
         })
     }
+    if (!is.null(modelRunState$optim)) {
```

Comment:
At the moment, if outputs become stale we just set `modelRunState$state` to "stale", so as currently stands the stale outputs will still be included in the saved digest. To fix we could either a) change this condition to also check that `state != "stale"` or b) set `optim` to null when the output becomes stale. I don't have a strong preference as to which!